sql/core/src/test/resources/sql-tests/inputs/pivot.sql

```diff
@@ -297,3 +297,13 @@ PIVOT (
   sum(earnings)
   FOR course IN ('dotNET', 'Java')
 );
+
+-- correctly handle pivot columns with different cases
+SELECT * FROM (
+  SELECT course, earnings, "a" as a, "z" as z, "b" as b, "y" as y, "c" as c, "x" as x, "d" as d, "w" as w
+  FROM courseSales
+)
+PIVOT (
+  sum(Earnings)
```

Comment:
Nice catch, @mgaido91 .
Instead of adding this new test case, please change `sum(earnings)` to `sum(Earnings)` in line 297.

Reply:
You can update the comment at line 291, too.